server/models/update/update_account.sql

```diff
@@ -0,0 +1,3408 @@
+-- DESCRIPTION
+-- First I wrote a SQL request to select all accounts of PCGC accounting referencial, 
+-- these accounts have a property "is_ohada = 0" The following query allowed us 
```

Comment:
This is a good description.

Could you make this use a simple `UPDATE ... SELECT` query?  Adding a 3.4K line file to the repository seems like it could be avoided.